testdata/cloud/misc/nginx-pod.yaml

```diff
@@ -8,7 +8,7 @@ spec:
   containers:
     -
       name: "myfrontend"
-      image: "quay.io/openshifttest/nginx-alpine@sha256:04f316442d48ba60e3ea0b5a67eb89b0b667abf1c198a3d0056ca748736336a0"
+      image: "quay.io/redhattraining/hello-world-nginx"
```

Comment:
@miyadav this will fail in disconncted cluster

Reply:
Thanks @sunzhaohua2 updated this